plugins/webp-uploads/helper.php

```diff
@@ -239,12 +239,13 @@ function webp_uploads_generate_image_size( $attachment_id, $size, $mime ) {
  * Returns the attachment sources array ordered by filesize.
  *
  * @since 1.0.0
+ * @todo This function is not used anywhere.
```

Comment:
We should check for when this function was first introduced and when it became no longer used.

Reply:
It seems its use was removed in 94f359ebe9729cdae7e3f012df22a8ca031fffdb as part of #302, but the `webp_uploads_get_attachment_sources()` function wasn't removed as well. I guess we'll throw in a `@deprecated` and `_deprecated_function()`.

Reply:
I think it's better to maintain a separate file for deprecated functions?

Reply:
Yeah, sure... done in 511b670602e9a9a15a38fc3247101c4f52d72dd8 and 13dd2954039b0f2721771baee02de77a3ea7d1bd